### ISSUE TITLE
DOC: Outdated environment types in docs

### ISSUE THREAD

**Issue by alecandido** [airspeed-velocity/asv#1386]
Following the instructions in the docs I got the following error:

```py
· Unknown environment type 'virtualenv'. Allowed values based on existing plugins are ['existing', 'conda'].
· No environments selected
```

The reason should be that `virtualenv` has been renamed to `existing` (or even a deeper change than that).

https://github.com/airspeed-velocity/asv/blob/a779f0cdd261dbcfc7010341af62fcdb2ca09e2c/docs/source/using.rst?plain=1#L80-L81
https://github.com/airspeed-velocity/asv/blob/a779f0cdd261dbcfc7010341af62fcdb2ca09e2c/asv/environment.py#L1030-L1031

(I tried to look for existing issues in the tracker, I hope I haven't missed anything...)

**Comment by HaoZeke** [comment]
No, the problem is not with the docs, but the environment found. Is `virtualenv` present in the environment being used to run `asv`?

**Comment by HaoZeke** [comment]
It perhaps could be clearer that the list of existing plugins are not necessarily all the plugins ASV supports. e.g. A `mamba` plugin is also supported but not listed as an option unless `libmambapy` is present. A PR to the docs would be most helpful.

**Comment by alecandido** [comment]
> No, the problem is not with the docs, but the environment found. Is `virtualenv` present in the environment being used to run `asv`?

Ok, sorry, I wrongly inferred the error. No, I'm pretty sure `virtualenv` was not present, but I'll double-check (I always use a minimal environment per-project, providing the Python executable and few other things, and mostly use `python -m venv` to create the environment).

For the time being, I realized that `asv` was the wrong tool for my current needs (I just want to benchmark a few implementations in isolation, not tracking the history of a production package), so I switched to the simpler `pytest-benchmark`.
However, I hope to get back to `asv` for another project. Then, I will take the time to author a PR for the docs.
